app/models/user.rb

```diff
@@ -1,14 +1,16 @@
+# frozen_string_literal: true
+
 class User < ApplicationRecord
-	has_many :gardens, dependent: :destroy
-	has_many :garden_plants, through: :gardens
-  validates_presence_of :name, :email, :google_token
+  has_many :gardens, dependent: :destroy
+  has_many :garden_plants, through: :gardens
+  validates_presence_of :email, :google_token
```

Comment:
removes the validation of presence of `name` because when the `scope` of `googleoauth` is changed to include `calendar` read/write access rights, the response from google does not include the `name` field.